dom.go

```diff
@@ -1080,16 +1087,21 @@ func (w *window) Stop() {
 
 // TODO reuse util.EventTarget
 
-func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) func(o *js.Object) {
```

Comment:
Please don't move AddEventListener around, it makes the diff noisier than it needs to be. It was fine being above the RemoveEventListener function.